net/offpunk/Portfile

```diff
@@ -15,9 +15,9 @@ long_description    {*}${description}. The goal of Offpunk is to be able \
                     to synchronise your content once (a day, a week, a month) \
                     and then browse/organise it while staying disconnected.
 
-checksums           rmd160  6e4d8ecd7abc5f1f1962f7dcb47048c5c74fd02a \
-                    sha256  a9abf9cfccefd2eec7b7583f8a2f88e69507917b70776151b3e7af5dbea2faf8 \
-                    size    793294
+checksums           rmd160  cea543940b1d6b71df8f27c064468e56cfcfb6fa \
+                    sha256  58a6931dc15dcb43f4443a9b8feb7bcabc72a87d8c3cad758985b11a7928ea13 \
+                    size    798743
 
 python.default_version 312
```

Comment:
if possible this should be using Python 3.13 by now as that is MacPorts' default

Reply:
[feedparser](https://ports.macports.org/port/py313-feedparser/), [setproctitle](https://ports.macports.org/port/py313-setproctitle/) and [readability-lxml](https://ports.macports.org/port/py313-readability-lxml/) are not tested for Python 3.13.

Reply:
I just pushed commits to add python 3.13 support for all of those. Also updated `py-setproctitle` to a newer version.